cluster/ubuntu/util.sh

```diff
@@ -231,6 +231,11 @@ EOF
 }
 
 function create-kubelet-opts() {
+  if [ -n "$6" ] ; then
+      cni_opts=" --network-plugin=cni --network-plugin-dir=/etc/cni/net.d"
```

Comment:
Let play around with local-up-cluster.  If I can get comfortable with that reasonably quickly then I will cover it too.